src/shaders.rs

```diff
@@ -82,7 +82,7 @@ pub fn full_shaders(
     device: &Device,
     engine: &mut WgpuEngine,
     options: &RendererOptions,
-) -> Result<FullShaders, Error> {
+) -> FullShaders {
```

Comment:
Is there value in having this part of the change be a separate PR that can land sooner than the rest? (Like I ended up doing with #547 for another part as I didn' know it was covered here.)

Reply:
probably.. It has some unwraps in it so I was hoping to get discussion instead of removing the `Result` but.. silence